esx_service/vmdk_ops.py

```diff
@@ -563,8 +564,11 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
 
     Returns None (if all OK) or error string
     """
-
     vm_datastore = get_datastore_name(config_path)
+    result = auth.authorize(vm_uuid, vm_datastore, cmd, opts)
```

Comment:
Will this method throw if there is no DB or the DB isn't initialized - typical errors like trying to authorize a VM thats not in any tenant, for example (tenants aren't created, DB isn't there).

Effectively, if DB isn't up, the datastore itself isn't usable and none of the containers in the VM are going to be able to run.

Should log as this is a critical error.